### ISSUE TITLE
[FEATURE] Start Development Backlog & Triage Meetings for the Security Analytics plugin

### ISSUE THREAD

**Issue by ict-one-nl** [opensearch-project/security-analytics#337]
**Is your feature request related to a problem?**
We are starting to use this functionality and would like to think along and provide input to aid the development process.

**What solution would you like?**
I'm joining the security guys regularly on the Mondays for the Development Backlog & Triage Meeting - Security. I mentioned that it would be nice to have these for the security analytics as well. They indicated it would be valid to open an issue, so here we are :) I find these meetings very helpful and it's nice to interact with the team.

**What alternatives have you considered?**
-

**Do you have any additional context?**
-

**Comment by AWSHurneyt** [comment]
Hi @ict-one-nl ,
You're suggesting security analytics host a similar meeting series, correct? Do you have some ideas for agenda items during such meetings? I think the development team would be open to the idea.

**Comment by ict-one-nl** [comment]
Yeah sure, at the security side they go through all the backlog items, triage them, judge the viability, discuss if it's desireable etc. They also discuss RFC's for bigger changes. As an involved user I find it very useful to provide feedback on stuff that's impacting our workflows. For the security analytics plugin I would welcome the possibility to also ask some questions and talk about the documentation.

**Comment by praveensameneni** [comment]
@sandervandegeijn , we plan to start office hours for security-analytics on Wednesdays starting next week (4/17) at 09:00 PT. Would this time work as it maybe 18:00 CET (Netherlands)? Will share a meeting invite with agenda.

**Comment by sandervandegeijn** [comment]
Cool! I will try to attend if possible (on holiday atm). Time is fine for us as well, but we shouldn't be the deciding factor for the time :)

**Comment by praveensameneni** [comment]
Will communicate more broadly in the next few days.
